JavaScript/JavaScript.sublime-syntax

```diff
@@ -1029,7 +1029,7 @@ contexts:
           set:
             - meta_scope: meta.function-call.method.js
             - include: function-call-params
-        - match: \b(process)(?:(\.)(abort|chdir|cwd|exit|getgid|setgid|getuid|setuid|setgroups|getgroups|initgroups|kill|memoryUsage|nextTick|umask|uptime|hrtime))?\b
+        - match: \b(process)(?:(\.)(abort|chdir|cwd|disconnect|exit|[sg]et[e]?gid|[sg]et[e]?uid|send|[sg]etgroups|initgroups|kill|memoryUsage|nextTick|umask|uptime|hrtime))?\b
```

Comment:
`[sg]ete?[gu]id` works too and is perfectly readable to someone versed in regular expressions, imo.